app/views/handlers/individual_response.py

```diff
@@ -300,39 +312,54 @@ def block_definition(self) -> Mapping:
                         self._list_name
                     ),
                 },
-                "description": [
-                    lazy_gettext(
-                        "For someone to complete a separate census, we need to send them an individual access code."
-                    ),
-                    lazy_gettext("Select how to send access code"),
-                ],
+                "description": self._build_question_description(),
                 "answers": [
                     {
                         "type": "Radio",
                         "id": "individual-response-how-answer",
                         "mandatory": False,
                         "default": "Text message",
-                        "options": [
-                            {
-                                "label": lazy_gettext("Text message"),
-                                "value": "Text message",
-                                "description": lazy_gettext(
-                                    "We will need their mobile number for this"
-                                ),
-                            },
-                            {
-                                "label": lazy_gettext("Post"),
-                                "value": "Post",
-                                "description": lazy_gettext(
-                                    "We can only send this to an unnamed resident at the registered household address"
-                                ),
-                            },
-                        ],
+                        "options": self._build_handler_answer_options(),
                     }
                 ],
             },
         }
 
+    def _build_handler_answer_options(self):
+        handler_options = [
+            {
+                "label": lazy_gettext("Text message"),
+                "value": "Text message",
+                "description": lazy_gettext(
+                    "We will need their mobile number for this"
+                ),
+            }
+        ]
+        if not self.has_postal_deadline_passed:
+            handler_options.append(
+                {
+                    "label": lazy_gettext("Post"),
+                    "value": "Post",
+                    "description": lazy_gettext(
+                        "We can only send this to an unnamed resident at the registered household address"
+                    ),
+                }
+            )
+        return handler_options
+
+    def _build_question_description(self):
+        description = (
+            "It is no longer possible to receive an access code by post"
+            if self.has_postal_deadline_passed
+            else "Select how to send access code"
```

Comment:
These won't be translated, lazy_gettext won't evaluate the function till they are required - if you look at the pot file diff the string `Select how to send access code` is now missing